synapse/tests/test_lib_cell.py

```diff
@@ -416,6 +416,18 @@ async def test_cell_promote(self):
                     with self.raises(s_exc.BadConfValu):
                         await proxy.promote()
 
+    async def test_cell_anon(self):
+
+        with self.getTestDir() as dirn:
+            conf = {'auth:anon': 'anon'}
+            async with await s_cell.Cell.anit(dirn, conf=conf) as cell:
+                anon = await cell.addUser('anon')
+                await cell.auth.rootuser.setPasswd('secret')
+                host, port = await cell.dmon.listen('tcp://127.0.0.1:0')
+                async with await s_telepath.openurl('tcp://127.0.0.1/', port=port) as prox:
+                    info = await prox.getCellUser()
+                    self.eq(anon.get('iden'), info.get('iden'))
```

Comment:
is there a sad path test where the anonuser is not present but enabled?

Reply:
adding